guides/v2.1/cloud/architecture/cloud-architecture.md

```diff
@@ -85,8 +84,18 @@ For comparison, each plan includes the following infrastructure features and sup
       <td>Dedicated technical account management for the initial launch period, starting with your subscription until your initial site launch</td>
       <td>Dedicated technical account management for the initial launch period, starting with your subscription until your initial site launch</td>
     </tr>
+    <tr>
+      <td>Add-ons*</td>
```

Comment:
Should there also be an entry for B2B in the Pro column?

Reply:
I thought Pro comes with B2B. Is it an add-on at an extra cost?

Reply:
Oh, I think you're right.